google/cloud/spanner_v1/testing/interceptors.py

```diff
@@ -0,0 +1,41 @@
+from collections import defaultdict
```

Comment:
Is this file intended to be public? If it is used for testing purposes, should it be under the `tests` directory